src/layout/OrgLayout.tsx

```diff
@@ -10,10 +11,55 @@ const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
     const orgQuery = useQuery(['org', orgId], getOrg(orgId));
 
     return (
-        <div>
-            <h1>{ orgQuery.data.title }</h1>
+        <>
```

Comment:
The organization page layout (i.e. this component) should probably always contain the main header, right? Keep that question in mind, and lets look at how things are working right now, and another way to achieve the same thing with less code.

I really like the way you wrapped your `OrgLayout` in `DefaultLayout` in the `getLayout()` function of `OrgEventsPage`. It looks something like this IIRC:

```jsx
<DefaultLayout>
  <OrgLayout orgId={ props.orgId }>
    { page }
  </OrgLayout>
</DefaultLayout>
```

Lets dissect what that means. It means that the resulting HTML should be generated using the following logic (simplified, and not entirely true but enough for the purpose of the reasoning):

1. Generate the page HTML
2. Generate the `OrgLayout` HTML, but replace `props.children` with the result from 1
3. Generate the `DefaultLayout` HTML, but replace `props.children` with the result from 2

This means that instead of any one of the page, `OrgLayout` or `DefaultLayout` components, we *could* just duplicate the JSX from the component and put it right here. So if, for the sake of this example, all that `OrgLayout` does is to wrap `props.children` in an `<article>` along with a `<h1>`, it would look like this:

```jsx
const OrgLayout = props => {
  return (
    <article>
      <h1>Organization page!</h1>
      { props.children }
    </article>
  );
};
```

If that was the case, this would be (almost) identical to just using the `<article>` directly in `getLayout()`, like so:

```jsx
<DefaultLayout>
  <article>
    <h1>Organization page!</h1>
    { page }
  </article>
</DefaultLayout>
```

This method of moving code from a function into the call site (i.e. the place the function was called) is often called "inlining", and is the opposite of breaking something out into a function. And just like with regular functions, React components can be "inlined" too.

I'm bringing this up to make sure that you understand that if what we want to achieve is the same logic that you have already applied, where `OrgLayout` (or rather the contents of `OrgLayout`) is composed within `DefaultLayout` (or rather, it's contents), that doesn't necessarily mean that the component `OrgLayout` is wrapped in `DefaultLayout`.

Instead, because `OrgLayout` never appears standalone, you could move the wrapping of the two components into this file and this line (14) instead, and just wrap *the content of `OrgLayout`* with `DefaultLayout`, right here *inside `OrgLayout`*.

That way, you only need to use `OrgLayout` in your `getLayout()` functions, and everything from `DefaultLayout` will be automatically included.

If you want to make it slightly nicer and perhaps separate concerns more distinctly, you could move all of the JSX that is currently in this file to a separate component, e.g. `src/components/OrgHeader.tsx`, and use that here.

That has the added benefit of being much easier to test using testing frameworks, which is something that I think we should start doing very soon. 😊

Reply:
Thanks! I think I follow. `DefaultLayout` is now added to `OrgLayout` instead and I created a new component `OrgHeader` (unsure if the tab menu should be part och this component).